.github/workflows/ci.yml

```
@@ -0,0 +1,150 @@
+
+      - name: Install Playwright browsers
+        run: npx playwright install --with-deps chromium
+
+      - name: Build Docker image
+        run: docker build -f Dockerfile.prod -t community-modules-viewer:test .
+
+      - name: Run E2E tests
+        run: |
+          # Start the container
+          docker run -d --name test-container -p 3002:80 community-modules-viewer:test
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)